react/common/components/returnList/JumpToPage.tsx

```diff
@@ -0,0 +1,61 @@
+import type { FormEvent } from 'react'
+import React, { useState } from 'react'
+import { FormattedMessage } from 'react-intl'
+import { Input, Button } from 'vtex.styleguide'
+
+interface Props {
+  handleJumpToPage: (page: number) => void
+  currentPage: number
+  maxPage: number
+}
+
+const JumpToPage = (props: Props) => {
+  const { handleJumpToPage, currentPage, maxPage } = props
+
+  const [desiredPage, setDesiredPage] = useState(0)
```

Comment:
In terms of UX, I think it would be better to initiate the `desiredPage` as the currentPage. This way, if I jump to page 2 then navigate to page one using the previous page button, they values will be aligned. Or every time that current page changes, it goes back to zero, or null. What do you think?

Reply:
Done